### DISCUSSION TITLE
Sending money with BTCPay via API

### DISCUSSION THREAD

**Discussion by ilearnio** [btcpayserver/btcpayserver#4737] [Technical Support]
I am looking for a way to allow our customers to withdraw their money (that were deposited previously) so that they can enter a custom amount and a BTC wallet address and the funds will be tranfered from our BTCPay. Is there any way to do that via API?

**Comment by Kukks** [comment]
https://docs.btcpayserver.org/PullPayments/#additional-use-cases-for-the-pull-payments-feature

**Comment by ilearnio** [comment]
@Kukks Thanks, that's helpful for sure. But I don't see an option to send the money automatically via API, without manually approving the payout and clicking on "Send selected payouts" in the BTCPay admin. Is there any way to approve and send via API?

**Comment by Kukks** [comment]
You can create pull payments with a flag for auto approval. Additionally we
have the concept of automated payout processors which allow scanning for
payouts awaiting payments and processing them in intervals. Alternatively,
theres also an opt in coinjoin plugin that you can install that will batch
these payouts in coinjoins for enhanced privacy.

**Comment by ilearnio** [comment]
Awesome! I will try using the flag for auto approval + payment processors. Hope it will work :)
Thank you so much!

**Comment by Garynieto38** [comment]
As this would be possible your asking the right questions just the wrong
person I don't have the answer
